treeherder/model/management/commands/test_analyze_perf.py

```diff
@@ -74,10 +74,15 @@ def handle(self, *args, **options):
                 signature_data = pc.get_performance_signatures(
                     project, time_interval=options['time_interval'])
                 signatures = []
+                signatures_should_ignored = []
```

Comment:
Let's make this a set, so we don't have tons of duplicate elements. Also, let's call this `signatures_to_ignore`. So:

    signatures_to_ignore = set()